### ISSUE TITLE
Extension doesn't work with CODE OWNERS feature

### ISSUE THREAD

**Issue by RussKie** [berzniz/github_pr_tree#110]
Hi there,
Looks like GitHub updated the markup again and your wonderful extensions isn't working

```
Uncaught TypeError: Cannot read property 'parentElement' of null
    at u (better_github_prs.js:1)
    at s (better_github_prs.js:1)
    at better_github_prs.js:1
    at Array.forEach (<anonymous>)
    at better_github_prs.js:1
    at Array.forEach (<anonymous>)
    at b (better_github_prs.js:1)
    at n.value (better_github_prs.js:31)
    at n.value (better_github_prs.js:31)
    at La (better_github_prs.js:22)
```

**Comment by mychaelgo** [comment]
looks like it's not working on private repository, but working on public repository 🤔

**Comment by RussKie** [comment]
Nop, https://github.com/dotnet/winforms/pull/3064/files
![image](https://user-images.githubusercontent.com/4403806/80772762-1e22a480-8b9b-11ea-8ed3-a83974513ccc.png)

Microsoft Edge: Version 81.0.416.68 (Official build) (64-bit)

**Comment by mychaelgo** [comment]
i see, i check that broken for GitHub Enterprise, looks normal for Github team and personal.

**Comment by mychaelgo** [comment]
@RussKie already create PR here https://github.com/berzniz/github_pr_tree/pull/111
Once it release by @berzniz , this issue should be solved. I face the same problem before.

**Comment by berzniz** [comment]
Thanks for the fix, I'll try to get it out as soon as possible.

As always, you can install the branch with the fix until then

**Comment by berzniz** [comment]
Will be fixed in 1.0.30 (deploying today to the Chrome/Firefox web stores)

**Comment by mychaelgo** [comment]
thanks @berzniz
